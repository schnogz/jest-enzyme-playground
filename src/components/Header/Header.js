import React from 'react';
import HoverLink from './../HoverLink/HoverLink';
import styled from 'styled-components';

const Wrapper = styled.section`
  margin: .4em;
  padding: 2em;
  background: papayawhip;
  text-align: center;
`;

const Header = () => (
	<Wrapper>
		<HoverLink page="https://www.github.com/schnogz/jest-enzyme-playground.git">Jest Enzyme Test Playground</HoverLink>
	</Wrapper>
);

export default Header;
