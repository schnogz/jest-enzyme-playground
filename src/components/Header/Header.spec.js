import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json';

import Header from './Header'

describe('Header component', () => {
  it('renders correctly', () => {
    const component = shallow(<Header />);
    const tree = toJson(component);
		expect(tree).toMatchSnapshot();
    expect(component.contains('Jest Enzyme Test Playground')).toBe(true);
  })
});