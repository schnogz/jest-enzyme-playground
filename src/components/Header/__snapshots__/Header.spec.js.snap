// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<styled.section>
  <HoverLink
    page="https://www.github.com/schnogz/jest-enzyme-playground.git"
  >
    Jest Enzyme Test Playground
  </HoverLink>
</styled.section>
`;
